src/Components/Members/Edit/EditMembers.jsx

```diff
@@ -49,20 +63,82 @@ const MembersEdit = () => {
     }
   ];
 
+  const schema = Joi.object({
+    firstName: Joi.string()
+      .pattern(/^[A-Za-z]+$/)
+      .min(3)
+      .max(15)
+      .messages({
+        'string.pattern.base': 'First name must contain only letters'
+      }),
+    lastName: Joi.string()
+      .pattern(/^[A-Za-z]+$/)
+      .min(3)
+      .max(15)
+      .messages({
+        'string.pattern.base': 'Last name must contain only letters'
+      }),
+    dni: Joi.string()
+      .pattern(/^[0-9]+$/)
+      .min(7)
+      .max(8)
+      .messages({
+        'string.pattern.base': 'DNI must contain only numbers'
+      }),
+    phone: Joi.string()
+      .pattern(/^[0-9]+$/)
+      .min(10)
+      .max(10)
+      .messages({
+        'string.pattern.base': 'Phone number must contain only numbers'
+      }),
+    email: Joi.string()
+      .max(30)
+      .pattern(/^[^@]+@[^@]+\.[a-zA-Z]{2,}$/)
+      .messages({
+        'string.pattern.base': 'Invalid email format'
+      }),
+    city: Joi.string().min(2).max(30),
+    birthDay: Joi.date().min('1930-01-01').max('2008-01-01'),
```

Comment:
The birth validation isn't working well, the input doesn't says the error and instead I received a modal 
![image](https://github.com/BaSP-m2023/ferpa-megarocket-app/assets/93209765/f1534eb4-967f-4fb7-8420-3b62674c8bbd)

Reply:
Also, if you are using the joi form errors, it shouldn't be a modal